src/status_im2/contexts/communities/actions/community_options/view.cljs

```diff
@@ -105,8 +105,7 @@
   {:icon                :i/edit
    :label               (i18n/label :t/edit-community)
    :accessibility-label :edit-community
-   :on-press            #(rf/dispatch [:communities/open-edit-community id]
-                                      (rf/dispatch [:hide-bottom-sheet]))})
+   :on-press            #(js/alert (str "implement action" id))})
```

Comment:
It would be nice if we could make a standardized alert for these features so the message/ui is always the same. Maybe we could start with this pr as it's small in scope, similar to how we are using the red border in the app

Reply:
@J-Son89, I don't mind. For me, the action id in that alert doesn't look necessary. We can use something simple like "not implemented". Wdyt?
cc @qoqobolo

Reply:
I guess it doesn't really matter what we use if we have a common function /component as we then only have to update it in one place 😀 
That sounds good to me 👍